iphone/Classes/TiUITableView.m

```diff
@@ -148,7 +154,11 @@ -(void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
 -(void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
 {
     if ([proxy _hasListeners:@"touchcancel"]) {
-        [proxy fireEvent:@"touchcancel" withObject:[proxy createEventObject:nil] propagate:YES];
+        
+        UITouch *touch = [touches anyObject];
+        CGPoint point = [touch locationInView:self];
+        NSDictionary *evt = [TiUtils touchPropertysToDictionary:touch:[touch locationInView:self]];
+        [proxy fireEvent:@"touchstart" withObject:evt propagate:YES];
```

Comment:
Do not copy and paste code. In this example, the "touchstart" event would be fired if the "touchcancel" listener is added.